WordPress/Classes/ViewRelated/System/Action Sheet/ActionSheetViewController.swift

```diff
@@ -24,6 +24,7 @@ class ActionSheetViewController: UIViewController {
         static let buttonSpacing: CGFloat = 8
         static let additionalSafeAreaInsetsRegular: UIEdgeInsets = UIEdgeInsets(top: 20, left: 0, bottom: 20, right: 0)
         static let minimumWidth: CGFloat = 300
+        static let maximumWidth: CGFloat = 600
```

Comment:
I based this on some documentation from Apple: https://developer.apple.com/documentation/uikit/uiviewcontroller/1619323-contentsizeforviewinpopover

> The recommended width for popovers is 320 points. If needed, you can return a width value as large as 600 points, but doing so is not recommended.

That property is deprecated now so the documentation might not be up to date but I couldn't find any other guidelines.